README.md

```diff
@@ -44,7 +44,7 @@ grub> terminal_input usb_gamepad
 
 ```console
 $ lsusb
-$ sudo qemu-system-i386 -usb -device usb-host,hostbus=<gamepad-bus>,hostaddr=<gamepad-addr> -cdrom test.iso
+$ sudo qemu-system-i386 -usb -device usb-host,vendorid=<gamepad-vendorid>,productid=<gamepad-productid> -cdrom test.iso
```

Comment:
Don't remove the original command. Add it as an alternative.

Reply:
I added it as a comment, so you can switch between both method of referencing.